netbout-client/src/main/java/com/netbout/client/RtInbox.java

```diff
@@ -126,7 +126,9 @@ public Bout bout(final long number) {
     public Iterable<Bout> search(final String term) throws IOException {
         final List<Bout> result = new ArrayList<>(16);
         for (final Bout bout : this.iterate()) {
-            if (bout.messages().search(term).iterator().hasNext()) {
+            if (bout.title().contains(term)) {
+                result.add(bout);
+            } else if (bout.messages().search(term).iterator().hasNext()) {
```

Comment:
@CarlosGines why an else here? Wouldn't it be better to have all the results? The above will search for bout titles and if found it won't look into messages.